### PR TITLE
Create entuty of car (#1)

### PR DESCRIPTION
**Pull request by OlegNovatskiy**
In it pull request was created car entity.

### CODE REVIEW

File: src/main/java/com/shopcar/entity/Car.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	public Car(){
+		
+	}
+	
+	public Car(String nameCar, String modelCar, Float engineCar){
+		this.nameCar = nameCar;
+		this.modelCar = modelCar;
+		this.engineCar = engineCar;
+	}
+
```
Comment (CortesV) [inline src/main/java/com/shopcar/entity/Car.java]: You need two constructors???????

Comment (ZVIproject) [inline src/main/java/com/shopcar/entity/Car.java]: Java Doc for constructors

Comment (OlegNovatskiy) [inline src/main/java/com/shopcar/entity/Car.java]: Yes, they should be required

Comment (OlegNovatskiy) [inline src/main/java/com/shopcar/entity/Car.java]: In next pull request it will be

**Comment by OlegNovatskiy** [comment]
JavaDoc was create)

**Comment by CortesV** [comment]
Approve

**Comment by ZVIproject** [comment]
Approve

**Pull request merged**